### PR TITLE
fix: raise config warning issue when masterIssue=true when bitbucket|azure

### LINKED ISSUE #3714: Massage Master Issue for Bitbucket and Azure DevOps
The Master Issue doesn't work on Bitbucket/Azure DevOps because:
- No HTML support for hidden data
- No dynamic checkbox support

Right now it can look like:
image

Let's try to make a workable "read only" master issue for these platforms, until one day there is the possibility to make an interactive one.

### PR DESCRIPTION
**Pull request by psyb0t**
Closes #3714

### CODE REVIEW

File: lib/workers/repository/master-issue.js [hunk 1]
```diff
@@ -1,8 +1,6 @@
 const { appName } = require('../../config/app-strings');
 
-module.exports = {
-  ensureMasterIssue,
-};
+module.exports = { ensureMasterIssue };
```
Comment (rarkins) [inline lib/workers/repository/master-issue.js]: Revert these changes

Comment (psyb0t) [reply lib/workers/repository/master-issue.js]: done

**Comment by rarkins** [comment]
@psyb0t I changed my mind and rewrote #3714 requirements, sorry! Let's start over.
